lib/init/features/functions/javascript.js

```diff
@@ -23,7 +23,7 @@ module.exports = function(setup, config) {
     }
   ]).then(function() {
     if (setup.functions.lint) {
-      _.set(setup, 'config.functions.predeploy', ['npm --prefix $RESOURCE_DIR run lint']);
+      _.set(setup, 'config.functions.predeploy', ['npm --prefix "$RESOURCE_DIR" run lint']);
```

Comment:
Please remove the double quotes around the env variable. 

Having the parenthesis around the env variable causes the script to break in Windows on powershell. Although the replacement works ("$RESOURCE_DIR" => "%RESOURCE_DIR%"), due to the quotes, powershell will interpret the env variable as a literal string. Without the double quotes, windows will replace the env variable with the actual value.